src/client.vala

```diff
@@ -87,9 +87,10 @@ private void on_subscribe_waybar (uint count, bool dnd, bool cc_open) {
         _class = "[%s, \"cc-open\"]".printf (_class);
     }
 
+    string total = "%u".printf (count);
     print (
-        "{\"text\": \"\", \"alt\": \"%s\", \"tooltip\": \"%s\", \"class\": %s}\n",
-        state, tooltip, _class);
+        "{\"text\": \"%s\", \"alt\": \"%s\", \"tooltip\": \"%s\", \"class\": %s}\n",
+        total, state, tooltip, _class);
```

Comment:
The previous code was fine, the linter was complaining about a missing space between the `to_string` and `();`